Documentation/CrossPlatform/OculusQuestMRTK.md

```diff
@@ -1,13 +1,18 @@
 # How to configure Oculus Quest in MRTK using the XRSDK pipeline
 
-A [Oculus Quest](https://www.oculus.com/quest/?locale=en_US) is required to use this data provider.
+A [Oculus Quest](https://www.oculus.com/quest/?locale=en_US) is required.
 
-The Oculus XRSDK Data Provider enables the use of the Oculus Quest with MRTK using [Unity's XR Pipeline](https://docs.unity3d.com/Manual/XR.html).
+MRTK's support for the Oculus Quest comes in the form of two data providers, the **Oculus XRSDK Data Provider** and the **MRTK-Quest** data provider. 
+
+The **Oculus XRSDK Data Provider** enables the use of the Oculus Quest with MRTK via the [Unity's XR Pipeline](https://docs.unity3d.com/Manual/XR.html).
 This pipeline is the standard for developing XR applications in Unity 2019.3 and beyond. To use this pipeline, make sure that you using **Unity 2019.3 or newer**
-The Oculus Quest XR Plugin does not currently support hand tracking. In order to use hand tracking, consult the [MRTK-Quest](https://github.com/provencher/MRTK-Quest)
-project which uses the Legacy XR pipeline.
+Specifically, this pipeline currently allows for the use of Oculus Touch controllers and head tracking with the Oculus Quest.
+
+The **MRTK-Quest Data Provider** enables the use of the Oculus Quest via the **Oculus Integration** Unity package. Specifically, this data provider has the additional benefit of 
+supporting **hand tracking** with the Oculus Quest. This data provider does **NOT** use the **XRSDK pipeline**, so it is recommended to set up this data provider after the 
```

Comment:
Wait should this read does not use the  **XRSDK pipeline** or  **legacy XR pipeline** - since it seems to be saying avoid legacy, it seems like this should mention the legacy one?

Reply:
Yeah I think I need to rewrite this section a bit now that we're pulling in MRTK-Quest directly into our project